### PR TITLE
Show warning for not supported distributions of Pennyworth 

### PR DESCRIPTION
**Pull request by vwallfahrer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ the environment of the integration tests.
 
 ## Installation
 
-Pennyworth is tested on [openSUSE 13.1](http://en.opensuse.org/Portal:13.1).
+Pennyworth is tested on [openSUSE 13.2](https://en.opensuse.org/Portal:13.2) and [SLES12].
```
Comment (tgoettlicher) [inline README.md]: Not everybody is familiar with the abbreviation SLES. Can you add a link to the homepage?
`[SUSE Linux Enterprise Server 12](https://www.suse.com/products/server/)`

Comment (tgoettlicher) [inline README.md]: It's confusion that it says `tested on 13.2` and below the installation instructions are for 13.1.

Comment (vwallfahrer) [inline README.md]: Should I replace the installation instructions header by 13.2?

Comment (tgoettlicher) [inline README.md]: Yes, if the installation instructions work for 13.2.

**Pull request closed without merging**